### PR TITLE
Migrate to chartkick

### LINKED ISSUE #1095: Load reports asynchronously
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**
The thing preventing us from using the reports endpoint right now is the page load timing out because of the heavy db load. One way around this is to load the reports asynchronously. We should do that!

* **What feature or behavior is this required for?**
Actually using reports.

* **How could we solve this issue? (Not knowing is okay!)**
I forget the most rails-y thing to do here is -- it might be setting up individual report endpoints and set up the individual partials to load with a `data-load`. Taker's choice!

* **Anything else?**
See also #1009

### PR DESCRIPTION
**Pull request by lapolonio**
I rule and have completed some work on Case Manager that's ready for review!

install chartkick and reproduce existing graphs

This pull request makes the following changes:
* install chartkick
* reformat JSON

(If there are changes to the views, please include a screenshot so we know what to look for!)

It relates to the following issue #s: 
* Fixes #1095

### CODE REVIEW

File: app/views/reports/_patient_report.html.erb [hunk 1]
```diff
@@ -43,7 +10,9 @@
   <h2 class="border-bottom">Year to date</h2>
   <% end %>
   <div class="row">
-    <div id='chart-<%= timeframe %>_report'></div>
+    <div id='chart-<%= timeframe %>_report'>
```
Comment (colinxfleming) [inline app/views/reports/_patient_report.html.erb]: Love how clean this turned out!

**Comment by lapolonio** [comment]
I didn't get it to work. Sorry I should have mentioned it was a work in progress. The payload is still an issue. I wanted an opinion on whether the manipulation should be in the query or frontend.

**Comment by colinxfleming** [comment]
No worries! 

If the current output of the report method doesn't work with chartkick, probably we should change the reporting method, since those are intended to smoothly create a product that fits into whatever charting software we're using. There's a related ticket in #1201 to do more filtering in DB query side, since the current strategy of 'get em all and then mapreduce em' is tough at any scale besides small data, so we're going to have to do some wrenching in there anyway. 

Do you have any interest in taking a whack at that? Happy to assign it out if not.

**Comment by colinxfleming** [comment]
We're going with a somewhat different reporting model now per @SerenaYudon 's recent work in #659 so closing this for the time being. My fault for the lousy guidance here, sorry bud.

**Pull request closed without merging**